backend/Cyberia/views.py

```diff
@@ -0,0 +1,102 @@
+from urllib import response
+from rest_framework.decorators import api_view
+from django.http import JsonResponse
+from dotenv import load_dotenv
+from rest_framework.response import Response
+from rest_framework import status
+from pathlib import Path
+import os
+import requests
+from Cyberia.NotionAPIparser import * 
+from threading import Thread
+
+dotenv_path = Path('backend/.env')
+load_dotenv(dotenv_path)
+token = os.getenv('token')
+version = os.getenv('version')
+components = dict()
+
+# Create your views here.
+
+@api_view(['Get'])
+def Cyberia(request):
+    data = []
+    
+    thread = [Thread(target=overview), Thread(target=benefit), 
+    Thread(target=time),Thread(target=registriation), Thread(target=trainers),
+    Thread(target=acadia_training), Thread(target=contact)]
+
+    for t in thread:
+        t.start()
+
+    for t in thread:
+        t.join()
+
+    result = dict()
+    data += components["overview"]
+    data += components["benefits"]
+    data += components["time"]
+    data += components["registriation"]
+    data += components["acadia_training"]
+    data += components["trainers"]
+    data += components["contact"]
+    result["result"] = data
+
+    return Response(data, status=status.HTTP_200_OK)
+
+
+def overview():
+    url = 'https://api.notion.com/v1/blocks/40712ecd5d6246aab46d9d51442c1727/children'
+    headers = {'Notion-Version': version, 'Authorization': token}
+    response = requests.get(url, headers=headers)
+    data = response.json()
+    components["overview"] = parse(data)
```

Comment:
Do you think it would be better to make all this functions (`overview()`, `benefit()`, etc) into just one function? Since if you notice, the only part that is different is only the `url` and specifically the `id` (after `/v1/blocks/`). Hence you can just make it a single function that takes `id` or `url` as an input, rather than having multiple functions that does the same thing. So then, for `overview()` you can just do `function("40712ecd5d6246aab46d9d51442c1727")`, for `benefit()` you can now do `function("1b95121ff4e24ab38c155c99c42675d9")`, and so on.

The aim for this is so that if you want to make a change, you only need to do it in one part/function, instead of all 7 functions, which would be tiring.